### ISSUE TITLE
Only showing funding source buttons

### ISSUE THREAD

**Issue by nsanden** [paypal/paypal-checkout#646]
It makes sense that you want to show the PayPal button along with the funding source buttons but i have the following situation.

I offer a Saas which is $x per year. For most users this works fine going through the typical buttons here:

![image](https://user-images.githubusercontent.com/2072064/38176079-fe19a284-359c-11e8-914a-f9d2784a687b.png)

Then, I setup a billing agreement and execute it server side, regardless of what button they click.

However some users do not want to have to create a PayPal account, or don't have a bank account attached to their PayPal account so billing agreements simply are not an option. I do not want to lose these potential customers though!

So for them I created another flow (same button as above) where if they click the credit card buttons, they will do a one time payment, not setting up any billing agreement, and this way they do not have to create a Paypal account. They can checkout as a guest. They do not have a subscription, but at least they have paid for 1 year and can use the product until then. At which time I will bug them to pay again.

The problem is I have to specifically tell them
1) hey, you need to go to this special checkout page on my site
2) you also need to make sure you click the credit card funding source buttons, NOT the paypal button

What should I do here?

Basically, i want to give them 3 options.

1) Pay with PayPal account
2) Pay with PayPal credit card (with or without paypal account)

Possible solutions:

1) One fix would be to allow me to only show the credit card buttons, not the PayPal button. This doesn't seem possible.
2) Another fix would be to allow me to initiate the checkout flow with my own button, but I have read in multiple github issues this is also not possible.
3) Last thing I can think of is if, inside payment() or onAuthorize(), if I could see WHICH button they clicked, then I could decide whether to create a billing agreement or one time payment. If they click the PayPal button, id create a billing agreement. If they click the credit card button, i'd create a one time payment. This also doesn't seem possible.

**Comment by bluepnume** [comment]
I think the ideal fix here us for us to enable guest billing agreements -- which I believe is something coming in the future. I'm going to mark this as a feature request, because we don't currently have plans to support options 1 through 3.

**Comment by nsanden** [comment]
Yes that seems like it would fix it, because the credit card buttons right now are kind of useless for a billing agreement setup. I've actually decided to remove them for now.

So now my flow looks like...

1) Show just PayPal payment button, with billing agreement setup
2) Once clicked, I show a "Having trouble checking out?" link below that button (in case they close the paypal popup windows early for whatever reason - then they will see this link and HOPEFULLY click it).
3) If a user clicks that Having trouble link, I reload the PayPal button with a new PayPal button and credit card buttons below it, and these are then all setup as one time payment instead of billing agreement.

This works, but so hacky for 2018, and i'm still losing customers (and getting support requests about having trouble paying).

Might have to try and come up with another flow entirely.
